modules/nodes-pods-autoscaling-custom-gather.adoc

```diff
@@ -0,0 +1,185 @@
+    ├── image.openshift.io
+    │   └── imagestreams.yaml
+    ├── k8s.ovn.org
+    │   ├── egressfirewalls.yaml
+    │   └── egressqoses.yaml
+    ├── keda.sh
+    │   ├── kedacontrollers
+    │   │   └── keda.yaml
+    │   ├── scaledobjects
+    │   │   └── example-scaledobject.yaml
+    │   └── triggerauthentications
+    │       └── example-triggerauthentication.yaml
+    ├── monitoring.coreos.com
+    │   └── servicemonitors.yaml
+    ├── networking.k8s.io
+    │   └── networkpolicies.yaml
+    ├── openshift-keda.yaml
+    ├── pods
+    │   ├── custom-metrics-autoscaler-operator-58bd9f458-ptgwx
+    │   │   ├── custom-metrics-autoscaler-operator
+    │   │   │   └── custom-metrics-autoscaler-operator
+    │   │   │       └── logs
+    │   │   │           ├── current.log
+    │   │   │           ├── previous.insecure.log
+    │   │   │           └── previous.log
+    │   │   └── custom-metrics-autoscaler-operator-58bd9f458-ptgwx.yaml
+    │   ├── custom-metrics-autoscaler-operator-58bd9f458-thbsh
+    │   │   └── custom-metrics-autoscaler-operator
+    │   │       └── custom-metrics-autoscaler-operator
+    │   │           └── logs
+    │   ├── keda-metrics-apiserver-65c7cc44fd-6wq4g
+    │   │   ├── keda-metrics-apiserver
+    │   │   │   └── keda-metrics-apiserver
+    │   │   │       └── logs
+    │   │   │           ├── current.log
+    │   │   │           ├── previous.insecure.log
+    │   │   │           └── previous.log
+    │   │   └── keda-metrics-apiserver-65c7cc44fd-6wq4g.yaml
+    │   └── keda-operator-776cbb6768-fb6m5
+    │       ├── keda-operator
+    │       │   └── keda-operator
+    │       │       └── logs
+    │       │           ├── current.log
+    │       │           ├── previous.insecure.log
+    │       │           └── previous.log
+    │       └── keda-operator-776cbb6768-fb6m5.yaml
+    ├── policy
+    │   └── poddisruptionbudgets.yaml
+    └── route.openshift.io
+        └── routes.yaml
+----
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
++
+[NOTE]
+====
+If your cluster is using a restricted network, you must take additional steps. If your mirror registry has a trusted CA, you must first add the trusted CA to the cluster. For all clusters on restricted networks, you must import the default `must-gather` image as an image stream.
+
+[source,terminal]
+----
+$ oc import-image is/must-gather -n openshift
+----
+====
+
+. Perform one of the following:
+
+* To get only the Custom Metrics Autoscaler Operator `must-gather` data, use the following command: 
++
+[source,terminal]
+----
+$ oc adm must-gather --image="$(oc get packagemanifests openshift-custom-metrics-autoscaler-operator \
+-n openshift-marketplace \
+-o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc. annotations.containerImage}')"
```

Comment:
In this step, and where this command is included up top (L30), there is a space that is not in L163 below. I am guessing it should be without the space?

```suggestion
-o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc.annotations.containerImage}')"
```

Reply:
@jeana-redhat Thanks for your great comments. I had two fairly different new must-gather features in two different OCP components and it got me all confused between the two!